Bio/File.py

```diff
@@ -472,10 +332,10 @@ def _load_index(self):
         index_filename = self._index_filename
         relative_path = self._relative_path
         filenames = self._filenames
-        format = self._format
+        fmt = self._format
         proxy_factory = self._proxy_factory
 
-        con = _sqlite.connect(index_filename, check_same_thread=False)
+        con = sqlite3.dbapi2.connect(index_filename, check_same_thread=False)
```

Comment:
Why ``sqlite3.dbapi2.connect`` versus ``sqlite3.connect``?

Reply:
original code has
```
     from sqlite3 import dbapi2 as _sqlite
```

Reply:
Hah. As you said, obfuscated.